src/migrations/20240610110823-remove-unwanted-objectives.js

```diff
@@ -0,0 +1,47 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      await prepMigration(queryInterface, transaction, __filename);
+      await queryInterface.sequelize.query(/* sql */`
+      -- Delete ARO's.
+      DELETE FROM "ActivityReportObjectives" WHERE "objectiveId" IN (
+        SELECT id FROM "Objectives" WHERE id IN (
+            179385,
+            179427,
+            179494,
+            179500,
+            179550,
+            179606,
+            179635,
+            179556,
+            180440,
+            177557) AND "status" = 'Not Started'
+      );
+
+        -- Delete objectives.
+        DELETE FROM "Objectives" WHERE id IN (
+          179385,
+          179427,
+          179494,
+          179500,
+          179550,
+          179606,
+          179635,
+          179556,
+          180440,
+          177557) AND "status" = 'Not Started';
```

Comment:
And there as no records in the Resource, Topic, Course, & File associated tables that link to any of these objectives? I would prefer that a delete for each of the link tables.

Reply:
Additionally, confirm that they are only on Draft reports as part of the migration

Reply:
I added ARO Topics and Resources.

I will also add a check for draft.